backend/Makefile

```diff
@@ -7,31 +7,43 @@ build:
 
 run:
 	-pkill docker-compose
-	DATABASE=postgres docker-compose up
+	docker-compose up postgres backend
+	docker-compose up
 
 run_background:
 	-pkill docker-compose
-	DATABASE=postgres docker-compose up -d
+	docker-compose up -d postgres backend
 
 stop:
 	docker-compose down
 
+run_test:
+	docker build -t rafflebay/test .
+	docker-compose up -d postgres_test test 
+	docker-compose exec postgres_test psql -p 5433 -U postgres rafflebay_test -c "DROP TABLE bids, items, users CASCADE;"
```

Comment:
Not sure what this command does but we also need to drop payments and shipments tables

Reply:
Removed make script